pkg/api/aim/runs.go

```diff
@@ -1042,5 +1043,6 @@ func toNumpy(values []float64) fiber.Map {
 
 // validateMetricNamePresent scans the query for metric.name condition
 func validateMetricNamePresent(query string) bool {
-	return strings.Contains(query, "metric.name == \"")
+	re := regexp.MustCompile(`(in\s)?metric.name[\.|(\s?==)]+`)
```

Comment:
The regex is a great idea! However I see a few issues with it:
* it won't match multiple whitespaces in `metric.name  == "accuracy"`
* it will match `metricsname`
* you use a character class (`[]`) so it will match all kinds of crazy things like `metric.name|`
* you force something to be behind `metric.name` so this won't match: `"accuracy" in metric.name`

I suggest the following regex instead — it may not be very elegant but I think it does the job!
```suggestion
	re := regexp.MustCompile(`in\s*metric\.name|metric\.name(?:\.|\s*==)`)
```

Reply:
also please make it a module var so it gets compiled only once!

Reply:
made these changes!